src/thorin/continuation.h

```diff
@@ -131,15 +124,9 @@ class Continuation : public Def {
     Intrinsic intrinsic() const { return attributes().intrinsic; }
     CC cc() const { return attributes().cc; }
     void set_intrinsic(); ///< Sets @p intrinsic_ derived on this @p Continuation's @p name.
-    void make_external() { attributes().visibility = Visibility::External; }
-    void make_internal() { attributes().visibility = Visibility::Internal; }
     bool is_basicblock() const;
     bool is_returning() const;
     bool is_intrinsic() const { return attributes().intrinsic != Intrinsic::None; }
-    bool is_external() const { return attributes().visibility == Visibility::External; }
-    bool is_internal() const { return attributes().visibility == Visibility::Internal; }
-    bool is_imported() const { return is_external() && empty(); }
-    bool is_exported() const { return is_external() && !empty(); }
```

Comment:
I would revert this: this distinction (what is exported/imported) is important to get the codegen right for C interfaces and probably in other places. We can keep perhaps the "externalness" as a property of the world, but we still need those helpers that combine it with empty/not_empty checks.

Reply:
libvulkan_radeon.so

Reply:
this also breaks the msvc path in continuation.cpp

Reply:
This was in fact one of the areas where I was a bit unsure about, but I had the impression that the distinction between external and exported/imported isn't consistently made everywhere, e.g. there are a some places where we check for `is_external()` and in addition for `empty()` instead of just checking for `is_imported()/is_exported`.  But now looking at this again, the distinction actually does make sense. I'd suggest to keep the external/internal stuff in the world and add the `is_imported`/`is_exported` getters back to `Continuation`s.